data_pipelines_cli/cli_commands/deploy.py

```diff
@@ -114,10 +117,15 @@ def _datahub_ingest() -> None:
                 "datahub",
                 "ingest",
                 "-c",
-                str(BUILD_DIR.joinpath("dag", "config", "base", "datahub.yml")),
+                str(self._find_datahub_config_file()),
```

Comment:
`datahub.yml` is also referenced in `compile.py` (function `_replace_datahub_with_jinja_vars`)